packages/fxa-shared/test/lib/sentry.ts

```diff
@@ -191,10 +191,10 @@ describe('lib/sentry', () => {
 
   describe('captureException', () => {
     it('calls Sentry.captureException', () => {
-      const captureException = sinon.spy(Sentry, 'captureException');
-      captureException(new Error('testo'));
-      sinon.assert.calledOnce(captureException);
-      captureException.restore();
+      const sentryCaptureException = sinon.stub(_Sentry, 'captureException');
```

Comment:
This test failed once it started getting run. Needed a tweak to properly mock `captureException` and call the *wrapper* rather than the *mock* as the test subject.